change state and change section settings gui and remote sync

lrealdi opened the pull request:
è possibile impostare da applicativo le configurazioni di change_state e di change_section che verranno salvate in ezsitedata 
è possibile eseguire un cambio stato su un oggetto o un cambio sezione su un nodo
le cinfigurazioni sono sincronuzzabili con istanze remote

(se necessario faccio una demo per visualizzare il funzionamento)

lrealdi committed:
change state and change section settings gui and remote sync

coppo reviewed (changes requested):
- Inserire confirm per "reset da file di  configurazione" e "Ripristina da ultimo backup"
- Fix interfaccia editing change section
- Aggiungere confirm su elimina regola/applicazione

lrealdi committed:
Inserire confirm per "reset da file di configurazione" e "Ripristina da ultimo backup"
Fix interfaccia editing change section
Aggiungere confirm su elimina regola/applicazione

lrealdi commented:
ping @coppo

lrealdi commented:
ping @coppo

coppo commented:
Aggiungerei qualche guida inline in fase di compilazione, tipo significato dei campi e sintassi.
A parte questo Squash & Lgtm
